src/main/java/utils/GcpClient.java

```diff
@@ -0,0 +1,45 @@
+package utils;
+
+import com.google.cloud.bigquery.*;
```

Comment:
Import specific classes that you need  instead of *

Reply:
Imported specific classes